pkg/sql/codegen/pai/kmeans.go

```diff
@@ -20,33 +20,73 @@ import (
 	"sqlflow.org/sqlflow/pkg/database"
 	"sqlflow.org/sqlflow/pkg/ir"
 	pb "sqlflow.org/sqlflow/pkg/proto"
+	"sqlflow.org/sqlflow/pkg/sql/codegen/attribute"
 )
 
+var kmeansAttributes = attribute.Dictionary{
+	"center_count": {attribute.Int, 3, `[default=3]
+The cluster count. range: [1, Infinity]
+`, attribute.IntLowerBoundChecker(1, true)},
+	"idx_table_name": {attribute.String, "", `
+The output table name which includes
+cluster_index column indicates the cluster result,
+distance column indicates the distance from the center and
+all the columns of input table.`, nil},
+	"excluded_columns": {attribute.String, "", `[default=""]
+excluded the special feature columns from the SELECT statment.`, nil},
```

Comment:
excluded -> Exclude
statment -> statement

Reply:
`excluded` is adj.  I think it's ok.